src/server/table_stats.h

```diff
@@ -1,187 +0,0 @@
-               total_incr_qps + total_check_and_set_qps + total_check_and_mutate_qps +
-               total_duplicate_qps;
-    }
-
-    double get_total_read_bytes() const
-    {
-        return total_get_bytes + total_multi_get_bytes + total_scan_bytes;
-    }
-
-    double get_total_write_bytes() const
-    {
-        return total_put_bytes + total_multi_put_bytes + total_check_and_set_bytes +
-               total_check_and_mutate_bytes;
-    }
-
-    void aggregate(const row_data &row)
-    {
-        total_get_qps += row.get_qps;
-        total_multi_get_qps += row.multi_get_qps;
-        total_put_qps += row.put_qps;
-        total_multi_put_qps += row.multi_put_qps;
-        total_remove_qps += row.remove_qps;
-        total_multi_remove_qps += row.multi_remove_qps;
-        total_incr_qps += row.incr_qps;
-        total_check_and_set_qps += row.check_and_set_qps;
-        total_check_and_mutate_qps += row.check_and_mutate_qps;
-        total_scan_qps += row.scan_qps;
-        total_duplicate_qps += row.duplicate_qps;
-        total_dup_shipped_ops += row.dup_shipped_ops;
-        total_dup_failed_shipping_ops += row.dup_failed_shipping_ops;
-        total_recent_read_cu += row.recent_read_cu;
-        total_recent_write_cu += row.recent_write_cu;
-        total_recent_expire_count += row.recent_expire_count;
-        total_recent_filter_count += row.recent_filter_count;
-        total_recent_abnormal_count += row.recent_abnormal_count;
-        total_recent_write_throttling_delay_count += row.recent_write_throttling_delay_count;
-        total_recent_write_throttling_reject_count += row.recent_write_throttling_reject_count;
-        total_storage_mb += row.storage_mb;
-        total_storage_count += row.storage_count;
-        total_rdb_block_cache_hit_count += row.rdb_block_cache_hit_count;
-        total_rdb_block_cache_total_count += row.rdb_block_cache_total_count;
-        total_rdb_index_and_filter_blocks_mem_usage += row.rdb_index_and_filter_blocks_mem_usage;
-        total_rdb_memtable_mem_usage += row.rdb_memtable_mem_usage;
-        total_rdb_estimate_num_keys += row.rdb_estimate_num_keys;
-        total_rdb_bf_seek_negatives += row.rdb_bf_seek_negatives;
-        total_rdb_bf_seek_total += row.rdb_bf_seek_total;
-        total_rdb_bf_point_positive_true += row.rdb_bf_point_positive_true;
-        total_rdb_bf_point_positive_total += row.rdb_bf_point_positive_total;
-        total_rdb_bf_point_negatives += row.rdb_bf_point_negatives;
-        total_backup_request_qps += row.backup_request_qps;
-        total_get_bytes += row.get_bytes;
-        total_multi_get_bytes += row.multi_get_bytes;
-        total_scan_bytes += row.scan_bytes;
-        total_put_bytes += row.put_bytes;
-        total_multi_put_bytes += row.multi_put_bytes;
-        total_check_and_set_bytes += row.check_and_set_bytes;
-        total_check_and_mutate_bytes += row.check_and_mutate_bytes;
-    }
-
-    void merge(const table_stats &row_stats)
```

Comment:
Could you please explain difference between function `aggeragate` and `merge` to me? I noticed that you use `aggerate` replace `merge` in info_collector.cpp.